assets/css/v2/triptych/screen_container.scss

```diff
@@ -13,5 +13,6 @@
 .left-pane,
 .middle-pane,
 .right-pane {
+  position: relative;
```

Comment:
It's a little confusing because it's spread across two files, but it is there so that each label positions absolutely within each pane and not the whole width of all panes. Without it, all 3 labels display on top of each other on the bottom right of the full screen. What I can do is move this into `evergreen.scss` so it's not so split up.